# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/src/APZCTreeManager.h
# Commit: 52ba9b55ca7c
# Full Hash: 52ba9b55ca7c15bb591db67c0573f3cb943739a6
# Author: Botond Ballo <botond@mozilla.com>
# Date: 2022-07-19 09:34:24
# Regressor Bug: 1779144
# File Overlap Count: 1
# Description:
#   Bug 1779144 - Factor out a helper shared by the touch event and tap event handling codepaths. r=dlrobertson
#   
#   Depends on D151939
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D151940
# ==============================================================================

diff -r fe20f47f85d9 -r 52ba9b55ca7c gfx/layers/apz/src/APZCTreeManager.h
--- a/gfx/layers/apz/src/APZCTreeManager.h	Mon Jul 18 21:37:47 2022 +0000
+++ b/gfx/layers/apz/src/APZCTreeManager.h	Mon Jul 18 21:37:48 2022 +0000
@@ -566,6 +566,9 @@
 
   ScreenMargin GetCompositorFixedLayerMargins() const;
 
+  void AdjustEventPointForDynamicToolbar(ScreenIntPoint& aEventPoint,
+                                         const HitTestResult& aHit);
+
   APZScrollGeneration NewAPZScrollGeneration() {
     // In the production code this function gets only called from the sampler
     // thread but in tests using nsIDOMWindowUtils.setAsyncScrollOffset this
